server/broadcast.go

```diff
@@ -448,6 +448,7 @@ func (bsm *BroadcastSessionsManager) shouldSkipVerification(sessions []*Broadcas
 }
 
 func NewSessionManager(ctx context.Context, node *core.LivepeerNode, params *core.StreamParameters, sel BroadcastSessionsSelectorFactory) *BroadcastSessionsManager {
+	params.Capabilities.SetMinVersionConstraint(node.Capabilities.MinVersionConstraint())
```

Comment:
Can `node.Capabilities` ever be null?

Reply:
I _think_ that no. But I'll add the IF just in case.